index.html

```diff
@@ -78,12 +88,12 @@ <h2 class="work">Work</h2>
       </div>
     </section>
     <section id="portfolio" class="works">
-      <h1 class="recent_heading">MY RECENT WORK </h1>
+      <h1 class="recent_heading"><span>M</span>Y RECENT WORK </h1>
     </section>
     <section id="About_myself" class="About_myself">
       <div class="project-container">
         <div class="image_placeholder"></div>
-        <div class="wrap-container">
+        <div class="wrap-container1">
           <h2 class="title_post">Multi-Post Stories</h2>
           <p class="supporting_text">
```

Comment:
- I see you try following the Figma design :+1: , exactly, but please try adding the missing background images as in the Figma design. 
##### Figma design 
![Screenshot from 2022-06-02 18-28-48](https://user-images.githubusercontent.com/58771507/171689585-d5b81641-4402-483d-88c7-fcb82f63a236.png)

##### Your design
![Screenshot from 2022-06-02 18-28-23](https://user-images.githubusercontent.com/58771507/171689686-edbee318-51e3-4234-934d-54e43caee611.png)

Reply:
background images added as requested.